Libraries/PeriphDrivers/Source/SPIXF/spixf_me55.c

```diff
@@ -41,14 +43,66 @@
 #include "spixf.h"
 #include "spixf_reva.h"
 
+#if defined(SPIXF_RAM)
+
+#include "gpio.h"
+#include "gpio_regs.h"
+#include "gpio_reva_regs.h"
+#include "mxc_pins.h"
+
+mxc_gpio_cfg_t gpio_cfg_spixf_ram = { MXC_GPIO1, (MXC_GPIO_PIN_22 | MXC_GPIO_PIN_23 | MXC_GPIO_PIN_24 |
+                                        MXC_GPIO_PIN_25 | MXC_GPIO_PIN_26 | MXC_GPIO_PIN_27), MXC_GPIO_FUNC_ALT1,
+                                        MXC_GPIO_PAD_NONE, MXC_GPIO_VSSEL_VDDIO, MXC_GPIO_DRVSTR_0 };
+
+#if IAR_PRAGMAS
+#pragma section = ".spix_config"
+#else
+__attribute__((section(".spix_config")))
+#endif
+int MXC_GPIO_Config_SPIXF(mxc_gpio_cfg_t *cfg)
```

Comment:
Might be a good idea to make this function static since the GPIO library follows the `MXC_GPIO...` naming and if this is not used anywhere outside this file.

Reply:
Fixed.